### PR TITLE
[W6.4h][W14-B2] Heng Zhi Xuan Caryn

### PR DESCRIPTION
**Pull request by hzxcaryn**
Done LO-Interfaces

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:3]: This interface and its methods should have header comments. 
If not, developers who need to implement this in their class will not know what the expected behavior is.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -63,4 +64,16 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
...
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
+        for (Printable printable : printables) {
+            builder.append(printable.getPrintableString());
+            builder.append(" "); // Whitespace btw each printable
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:74]: isn't it apparent when you append an empty space? you can remove such redundant comments.

**Comment by okkhoy** [comment]
@hzxcaryn 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/ui/Formatter.java (hunk 2 above)
Comment (hzxcaryn) [reply src/seedu/addressbook/ui/Formatter.java:74]: Ok noted!

**Pull request closed without merging**